contents/handbook/small-teams/marketing/index.mdx

```diff
@@ -13,9 +13,31 @@ hideAnchor: false
 
 <Mission />
 
-## Q4 2023 Goals
-
-<Objectives />
+## Q1 2024 Goals
+
+**Objective**
+Increase the rate of people activating into their 2nd and 3rd PostHog products.
```

Comment:
Nit: Marketing teams are better placed to increase acquisition, whereas growth team are better suited to increase activation rates once users have signed up to the product. So I think we should focus on increasing the "total number", not the "rate"

Also, we're not well placed to get people to start using their 2nd and 3rd products (and our projects dont quite align with that either). Instead I suggest replacing this with "non-analytics" products

```suggestion
Increase the number of people signing up for PostHog's non-analytics products.
```

Reply:
Good points! Will slightly tweak to clarify we mean Product Analytics specifically but otherwise agree